### PR TITLE
feat: implement 1H NMR endpoint

### LINKED ISSUE #1: implement 1H NMR endpoint

### PR DESCRIPTION
**Pull request by kjappelbaum**
Fixes #1

How do you want to handle unit tests? I feel it would be good to set up some. For this, it is perhaps easier if we have an installable package and then use pytest?

### CODE REVIEW

File: tools/cheminformatics/modal.py [hunk 1]
```diff
@@ -24,3 +23,42 @@ def tanimoto(s1: str, s2: str) -> float:
         return DataStructs.TanimotoSimilarity(fp1, fp2)
     except (TypeError, ValueError, AttributeError):
         return "Error: Not a valid SMILES string"
+
+
+@app.function
```
Comment (pschwllr) [inline tools/cheminformatics/modal.py]: Would you need `@app.function(image=rdkit_image)`?

File: tools/cheminformatics/modal.py [hunk 2]
```diff
@@ -1,17 +1,16 @@
 from modal import App, web_endpoint
 from modal import Image
 
-rdkit_image = (
-    Image.debian_slim(python_version="3.12")
-    .pip_install("rdkit")
-)
+rdkit_image = Image.debian_slim(python_version="3.12").pip_install(["rdkit", "numpy"])
```
Comment (pschwllr) [inline tools/cheminformatics/modal.py]: Would it be good to fix the versions of the packages? Numpy just had a major update (and at least in my course there were a lot of projects with breaking changes).

**Comment by whitead** [comment]
Hi @kjappelbaum - thanks for getting this! I fixed CI/deploy strategy and now you can rebase to main and get the code deployed/built to a "-dev" instance.

**Comment by kjappelbaum** [comment]
@whitead is there any chance you removed the secrets from this repo?

**Pull request closed without merging**